cmd/chainbridge/deploy.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"encoding/hex"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/ChainSafe/ChainBridgeV2/keystore"
+	log "github.com/ChainSafe/log15"
+	"github.com/urfave/cli"
+)
+
+var deployCommand = cli.Command{
+	Action:   handleDeployCmd,
+	Name:     "deploy",
+	Usage:    "deploys test ganache",
+	Category: "TESTING",
+	Flags:    DeployFlags,
+	Description: "\tThe deploy command is used to deploy a ganache instance for testing\n" +
+		"\tTo add additional accounts, -testaccount\n" +
+		"\tTo only use custom accounts, -reset\n" +
+		"\tTo use a mnemonic, -mnemonic",
+}
+
+// The Private Keys of Alice, Bob, Etc
+var BaseAccounts = []string{
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.AliceKey].Encode()),
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.BobKey].Encode()),
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.CharlieKey].Encode()),
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.DaveKey].Encode()),
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.EveKey].Encode()),
+}
+
+// deployArgd holds the args to run tests with ganache
+type deployArgs struct {
+	port     string
+	amount   string
+	mnemonic string
+	accounts []string
+}
+
+func newdeployArgs() *deployArgs {
+	return &deployArgs{
+		accounts: BaseAccounts,
+		port:     "8545",
+		amount:   "100000000000000000000",
+		mnemonic: "",
+	}
+}
+
+func handleDeployCmd(ctx *cli.Context) error {
```

Comment:
Please can we rename this, perhaps to `ganache`. `deploy` will presumably be what actually deploys the contracts, which should be done separately from starting ganache.

Reply:
I think `handleGanacheCmd` might be better to fit with other functions. Thoughts?

Reply:
Exactly, was just hinting at the other changes that this implies